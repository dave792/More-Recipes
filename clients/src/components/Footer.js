import React from 'react';
import '../css/style.css';


const Footer = () => (
  <footer>
      <hr />
        <div className="container">
            <div className="row">

                <div className="col-md-12">
                    <div className="text-center">
                        <p style={{color: '#2c3e50' }}>Vanessa Ating &copy; 2017. All Rights Reserved.</p>
                    </div>
                </div>
            </div>
        </div>

    </footer>
);

export default Footer;

