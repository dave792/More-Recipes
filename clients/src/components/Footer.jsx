import React from 'react';

const Footer = () => (
  <footer>
    <hr />
    <div className="container">
      <div className="row">

        <div className="col-md-12">
          <div className="text-center">

            <p style={{ color: '#2c3e50' }}>
            Vanessa Ating &copy; 2017. All Rights Reserved.
            </p>

          </div>
        </div>
      </div>
    </div>

  </footer>
);

export default Footer;

