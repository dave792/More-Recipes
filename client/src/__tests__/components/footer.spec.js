import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../../components/Footer';

describe('The Footer component', () => {
  it('Should mount the Footer component without errors', () => {
    const tree = renderer.create(<Footer />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
