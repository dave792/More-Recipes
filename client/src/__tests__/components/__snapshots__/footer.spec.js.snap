// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Footer component Should mount the Footer component without errors 1`] = `
<footer>
  <hr />
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <div
          className="text-center"
        >
          <p
            style={
              Object {
                "color": "#2c3e50",
              }
            }
          >
            Vanessa Ating Â© 2017. All Rights Reserved.
          </p>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
